io_scene_dae/export_dae.py

```diff
@@ -407,13 +414,31 @@ def export_mesh(self, node, armature=None, skeyindex=-1, skel_source=None,
                 shape.value = 1.0
                 mesh.update()
                 p = node.data
-                v = node.to_mesh(bpy.context.scene, True, "RENDER")
+                
+                armature_modifier = None            
+                armature_modifier_state = None
+                
+                if(self.config["use_exclude_armature_modifier"]):
+                    armature_modifiers = [i for i in node.modifiers if i.type == "ARMATURE"]
+                    armature_modifier = armature_modifiers[0]#node.modifiers.get("Armature")
```

Comment:
This line is causing an index out of bound exception when I try to export a simple box. _armature\_modifiers_ list can be empty, so it can't be directly accessed like this.